systest/testdata/manifest/apply.txtar

```diff
@@ -37,21 +37,21 @@ ak connection get my_project/my_connection -J
 output equals file expected_connection_details.json
 return code == 0
 
-ak connection get connection:00000000000000000000000000000002 -J
+ak connection get connection:00000000000000000000000000000003 -J
```

Comment:
```suggestion
# TODO: Add more tests - either here, or in separate `.txtar` files:
# - Specifying an env name
# - This command's flags
# - Other manifest commands since PR #6

ak connection get connection:00000000000000000000000000000003 -J
```

Reply:
done